docs/en/ingest-management/security/certificates.asciidoc

```diff
@@ -231,6 +231,7 @@ sudo ./elastic-agent install \
    --fleet-server-port=8220 \
    --elastic-agent-cert=/tmp/fleet-server.crt \
    --elastic-agent-cert-key=/tmp/fleet-server.key \
+   --elastic-agent-cert-key-passphrase=/tmp/fleet-server/passphrase-file \
```

Comment:
The comment thread seems to be gone, so just for reference I changed the `?????` to `passphrase-file` for clarity.